packages/front-end/components/Modal.tsx

```diff
@@ -148,17 +155,15 @@ const Modal: FC<ModalProps> = ({
             ""
           )}
           {secondaryCTA}
-          {close && (
```

Comment:
Still need this.  The `close` prop is optional and we only want to render the button if it's set.

Reply:
Doh! Updated this - I think it should be correct now?